src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -218,6 +221,34 @@ ScrollDown.propTypes = {
 
 export default function Banner() {
   const classes = useStyles();
+  const { telescopeUrl } = useSiteMetadata();
+  const [sha, setSha] = useState('');
+  const [version, setVersion] = useState('');
+  const [githubUrl, setGitHubUrl] = useState('');
+
+  async function getGitData() {
+    try {
+      const res = await fetch(`${telescopeUrl}/health`);
+
+      // Fetch failure
+      if (!res.ok) {
+        throw new Error(res.statusText);
+      }
+
+      const data = await res.json();
+      setSha(data.info.sha);
+      setGitHubUrl(data.info.gitHubUrl);
+      setVersion(data.info.version);
+    } catch (error) {
+      console.error(`Error retrieving site's health info`, error);
+      throw new Error('Error retrieving site health info', error);
+    }
+  }
+
+  useEffect(async () => {
+    await getGitData();
```

Comment:
You're not allowed to do this in a hook (they have to return `undefined` vs. a `Promise`).  Change it like this:

```js
useEffect(() => {
  async function getGitData() {...}

  getGitData();
});
```